packages/lib/gpcircuits/scripts/gen-circuit-parameters.ts

```diff
@@ -0,0 +1,95 @@
+import circomkitJson from "../circomkit.json";
+import { Circomkit } from "circomkit";
+import * as fs from "fs/promises";
+import { existsSync as fsExists } from "fs";
+import * as path from "path";
+import {
+  ProtoPODGPC,
+  ProtoPODGPCParameters,
+  PROTO_POD_GPC_PUBLIC_INPUT_NAMES
+} from "../src/proto-pod-gpc";
+import { batchPromise, clearDir, maxParallelPromises } from "../src/util";
+
+// Circuit parameters used to generate artifacts.
+const CIRCUIT_PARAMETERS = [
+  [1, 1, 5],
+  [1, 5, 8],
+  [3, 10, 8]
+];
+
+const circuitDir = path.join("circuits", "main");
```

Comment:
Make this absolute using __dirname.  I don't trust the cwd to always be the same in scripts like this.

Admittedly I'm also not sure whether the circom calls will be similarly robust.  I think it depends on cwd to find its config file.  You could load that explicitly, but it has relative paths inside it which may or may not work from another cwd anyway.  No need to fully tackle this for now, but I think that defaulting to absolute paths where possible is a good general practice.

Reply:
I just made a helper for loading and fixing Circomkit config file to use absolute paths, so it's easier to use it from outside the package: https://github.com/proofcarryingdata/zupass/pull/1662/files#diff-5bbef911a8cc882b32f2afc905cfd8c310fa6dc56bff1ce6031a11141c31d169R101